src/components/pages/Theme/Theme.jsx

```diff
@@ -15,6 +15,7 @@ import {
 
 // styles
 import './Theme.css';
+import '../../../ga.css';
```

Comment:
I think it's already in `index.js`, so you don't need to import it here.
```
// styles
import 'semantic-ui-css/semantic.css';
import './ga.css';
```

Reply:
It didn't work, I am taking a look on why was that